modules/vsphere-load-balancer-multiple-subnets.adoc

```diff
@@ -0,0 +1,119 @@
+// Module included in the following assemblies:
+//
+// * installing_vsphere/installing-vsphere-installer-provisioned-customizations.adoc
+// * installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.adoc
+// * installing_vsphere/installing-restricted-networks-installer-provisioned-vsphere.adoc
+// * installing_vsphere/installing-vsphere-installer-provisioned.adoc
+
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../installing/installing_vsphere/installing-vsphere-installer-provisioned.adoc[Installing a cluster on vSphere]
+
+[id="vsphere-load-balancer-mutliple-subnets_{context}"]
+= Configuring an external load balancer for multiple subnets
+
+You can configure an {product-title} cluster to use an external load balancer that supports multiple subnets.
+
+You must modify the default external load balancer by defining multiple network components, subnets, for vSphere. You can then distribute control plane nodes and compute nodes across multiple subnets to better manage scheduling of available pods.
+
+You do not need to specify API and Ingress static addresses for your installation program. If you choose this configuration, you must take additional actions to define network targets that accept an IP address from each referenced vSphere subnet.
```

Comment:
```suggestion
You do not need to define API and Ingress VIPs during installation.  However, the API and Ingress DNS A records must be defined and point to the external load balancer.
```

Reply:
Update.

Reply:
> f "DNS A" is a typo?

@dfitzmau Nope. Not a typo a DNS A record. This is a type of DNS record. The intended audience of this topic will no exactly what this is.

Reply:
Thanks. Updated.